source/standards/pull-requests.html.md.erb

```diff
@@ -1,156 +1,156 @@
 ---
 title: Pull requests
-last_reviewed_on: 2019-12-27
+last_reviewed_on: 2020-06-18
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
 ## Why should you use pull requests?
 
-- It's good practice to have a second opinion
-- It's part of our accreditation - the extra pair of (unconnected - even if you
-  pair you need a third person to merge) eyes provides a lightweight change
+We use pull requests at GDS because:
+
+- it's good practice to use them to [review code](manuals/code-review-guidelines.html)
```

Comment:
I think the url here needs a / at the front so /manuals/code-review-guidelines.html

Reply:
Ah yes! Will fix that now